src/gui/main.c

```diff
@@ -84,7 +84,7 @@ char_callback(GLFWwindow* window, unsigned int c)
 static void
 scroll_callback(GLFWwindow *window, double xoff, double yoff)
 {
-  dt_gui_imgui_scrolled(window, xoff, yoff);
```

Comment:
is this not called any more now? i suppose this causes trouble for std imgui widgets that interact via mouse wheel? does for instance light table still scroll down with this line removed?

i suppose we should just call both. there is a dt_gui_imgui_want_mouse() function to wrap imgui's mouse request, but it seems to be always on..

Reply:
> does for instance light table still scroll down with this line removed?

You are right. It doesn't work anymore.
Just added the line again.

Reply:
> i suppose we should just call both. there is a dt_gui_imgui_want_mouse() function to wrap imgui's mouse request, but it seems to be always on..

I haven't tested that (as it was commented in your code).